spec/spec_helper.rb

```diff
@@ -16,7 +16,14 @@
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 #
+require "require_all"
+
 require "aasm/rspec"
+
+# payment calculation code
+require_relative "../lib/initialize_with_config"
+require_all "lib/services"
```

Comment:
I don't think this warrants a gem. Can we autoload these?

Reply:
@jamesrose Hmm. Do you mean autoload?  Wouldn't it be better to require them explicitly if we don't like that testing gem. 

Setting the config.autoload_paths array seems a little heavy? I'm not even sure that's documented in the file anymore. I've got some experience of doing that, and it's normally quite hard for people to find unless they know how Rails internals work.

Reply:
Or do you mean to do it at the application level the same way that it's being done to force reloading of decorators and serializers? Scoped for test in application.rb?

Reply:
Ah. Lightbulb moment. It's just for Rspec, so I'm guessing you can do that just in the .rspec file.

Reply:
@jamesrose Are you happier with 3a593d9ec88a4954e1afe7b3399cb9d87336c8ef by @blackrat that eliminates it?
It does mean that there are lots more require statements around, which maybe is fine.

Reply:
@timabell @blackrat Yes, I think that's fine 👍